ReactApp/src/misc/Helpers.js

```diff
@@ -1,8 +1,43 @@
-export function postMessage(message, parameters = {}) {
-	if (window.webkit) {
-		const value = { message: message, body: parameters };
-		window.webkit.messageHandlers.editorDelegate.postMessage(value);
+export function editorLoaded() {
+	console.log("Firing JS editorLoaded event");
+
+	if(window.editorDelegate) {
+		editorDelegate.onEditorLoaded();
```

Comment:
Although by itself `editorDelegate` exists, to prevent ESLint formatting warnings, could we update this to:

```suggestion
		window.editorDelegate.onEditorLoaded();
```